File: koin/src/main/java/in/koreatech/koin/data/network/response/DefaultResponse.java
package in.koreatech.koin.data.network.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DefaultResponse {
    @SerializedName("success")
    @Expose
    public boolean success;

    @SerializedName("message")
    @Expose
    public String message;

    @SerializedName("error")
    @Expose
    public String error;

    @SerializedName("code")
    @Expose
    public String code;

    @SerializedName("grantEdit")
    @Expose
    public boolean isGrantEdit;


    public boolean getSuccess() {
        return success;
    }

    public String getMessage() {
        return message;
    }

    public String getError() {
        return error;
    }

    public String getCode() {
        return code;
    }

    public boolean isGrantEdit() {
        return isGrantEdit;
    }
}

File: data/src/main/java/in/koreatech/koin/data/response/DefaultResponse.kt
package `in`.koreatech.koin.data.response

import com.google.gson.annotations.SerializedName

data class DefaultResponse(
    val success: Boolean,
    val message: String,
    val error: String,
    val code: String,
    @SerializedName("grantEdit") val isGrantEdit: Boolean
)
